README.md

```diff
@@ -196,13 +216,13 @@ git add . && git commit -m "modify forest depth"
 git push origin experiment
 ```
 
-5. In GitHub, open up a Pull Request to compare the `experiment` branch to
+5. In GitHub, open up a pull request to compare the `experiment` branch to
```

Comment:
I find better Pull Request 🤔 its the name of the action actually. I mean... is a GH thing

Reply:
> [_**Pull requests** let you tell others about changes you've pushed to a branch in a repository on GitHub. Once a **pull request** is opened, you can discuss and review the potential changes with collaborators and add follow-up commits before your changes are merged into the base branch._](https://docs.github.com/es/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests)

If they don't capitalize it, I guess that we shouldn't.

Reply:
> If they don't capitalize it, I guess that we shouldn't.

This is interesting!!! 
Bitbucket seems to be the one capitalizing it.
A search in google also opens a holy war

Reply:
⚔️  I have a personal preference for _pull request_ without any creative capitalization. Unless I'm wrong, this is an action noun, like _disagreement_ or _discussion_, 😉  and it shouldn't be treated differently to any others. 😄